phpstan.neon.dist

```diff
@@ -17,6 +17,8 @@ parameters:
         - '~^Parameter #1 \$files of method Doctrine\\Migrations\\Finder\\Finder::loadMigrationClasses\(\) expects array<string>, array<int, string\|false> given\.\z~'
         - '~^Class Doctrine\\Migrations\\Tests\\DoesNotExistAtAll not found\.\z~'
         - '~^Call to method getVersion\(\) of deprecated class PackageVersions\\Versions\.$~'
+        # Drop when bumping to doctrine/dbal 2.11+
+        - '~^.*PrimaryReadReplicaConnection.*$~'
```

Comment:
phpstan is ran on 7.1, which prevents installing a recent enough version of `doctrine/dbal`, and generates quite a few errors because the class does not exist. The errors can be seen (for now) at https://github.com/doctrine/migrations/runs/1489165823?check_suite_focus=true